packages/discovery-provider/src/api/v1/models/users.py

```diff
@@ -155,6 +155,15 @@
     },
 )
 
+split = ns.model(
+    "purchase_split",
```

Comment:
I may be confused, but if we're storing the percentage in the DB, why are we not returning it in this model?

Reply:
Yeah these percentages are another thing I'd want to address. I purposely left them out because they're not useful right now in this flow anyway, and they can be misleading. For example, now with the network split, "100%" doesn't really look like "100%"

I think it's worth a conversation about whether we ultimately have "nested" percentages, like:
```
"splits": [
    {
        "label": "users", 
        "percentage": 90, 
        "splits": [
            { "user_id": 1, "percentage": 100, ... },
        ]
   },
   { "label": "network", "percentage": 100, ... }
]
```

or do we recalculate percentages after adding the network cut?

I should write up more details on this... 

but yeah i think it's out of scope for right now - we just want to see amounts for the receipts

Reply:
and to your point, it probably shouldn't be in the DB either then, but I didn't feel like adding yet another split format would be beneficial.